catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -22,6 +23,8 @@
 		<bean id="listConverter" class="ddf.catalog.util.impl.ListConverter"/>
 	</type-converters>
 
```

Comment:
The indentations in this file are all messed up still. Would suggest reformatting the whole thing and keeping that reformat in a separate commit.